examples/pincell_random_ray/build_xml.py

```diff
@@ -0,0 +1,211 @@
+
+water = openmc.Material(name='Water')
+water.set_density('macro', 1.0)
+water.add_macroscopic(h2o_data)
+
+# Instantiate a Materials collection and export to XML
+materials_file = openmc.Materials([uo2, water])
+materials_file.cross_sections = "mgxs.h5"
+materials_file.export_to_xml()
+
+###############################################################################
+# Define problem geometry
+
+# The geometry we will define a simplified pincell with fuel radius 0.54 cm
+# surrounded by moderator (same as in the multigroup example).
+# In random ray, we typically want several radial regions and azimuthal
+# sectors in both the fuel and moderator areas of the pincell. This is
+# due to the flat source approximation requiring that source regions are
+# small compared to the typical mean free path of a neutron. Below we
+# sudivide the basic pincell into 8 aziumthal sectors (pizza slices) and
+# 5 concentric rings in both the fuel and moderator.
+
+# TODO: When available in OpenMC, use cylindrical lattice instead to
+# simplify definition and improve runtime performance.
+
+pincell_base = openmc.Universe()
+
+# These are the subdivided radii (creating 5 concentric regions in the
+# fuel and moderator)
+ring_radii = [0.241, 0.341, 0.418, 0.482, 0.54, 0.572, 0.612, 0.694, 0.786]
+fills = [uo2, uo2, uo2, uo2, uo2, water, water, water, water, water]
+
+# We then create cells representing the bounded rings, with special
+# treatment for both the innermost and outermost cells
+cells = []
+for r in range(10):
+    cell = []
+    if r == 0:
+        outer_bound = openmc.ZCylinder(r=ring_radii[r])
+        cell = openmc.Cell(fill=fills[r], region=-outer_bound) 
+    elif r == 9:
+        inner_bound = openmc.ZCylinder(r=ring_radii[r-1])
+        cell = openmc.Cell(fill=fills[r], region=+inner_bound) 
+    else:
+        inner_bound = openmc.ZCylinder(r=ring_radii[r-1])
+        outer_bound = openmc.ZCylinder(r=ring_radii[r])
+        cell = openmc.Cell(fill=fills[r], region=+inner_bound & -outer_bound) 
+    pincell_base.add_cell(cell)
+
+# We then generate 8 planes to bound 8 azimuthal sectors
+azimuthal_planes = []
+for i in range(8):
+    angle = 2 * i * openmc.pi / 8
+    normal_vector = (-openmc.sin(angle), openmc.cos(angle), 0)
+    azimuthal_planes.append(openmc.Plane(a=normal_vector[0], b=normal_vector[1], c=normal_vector[2], d=0))
+
+# Create a cell for each azimuthal sector using the pincell base class
+azimuthal_cells = []
+for i in range(8):
+    azimuthal_cell = openmc.Cell(name=f'azimuthal_cell_{i}')
+    azimuthal_cell.fill = pincell_base
+    azimuthal_cell.region = +azimuthal_planes[i] & -azimuthal_planes[(i+1) % 8]
+    azimuthal_cells.append(azimuthal_cell)
+
+# Create the (subdivided) geometry with the azimuthal universes
+pincell = openmc.Universe(cells=azimuthal_cells)
+
+# Create a region represented as the inside of a rectangular prism
+pitch = 1.26
+box = openmc.model.RectangularPrism(pitch, pitch, boundary_type='reflective')
```

Comment:
```suggestion
box = openmc.model.rectangular_prism(pitch, pitch, boundary_type='reflective')
```

Reply:
Thanks for taking a look at this @shimwell! I think you're right, but the issue was happening in the test file rather than the example file. I think I've fixed it now, but am still working through some other issues with clang format. The new test is now running cleanly locally at least.